portals/publisher/source/src/app/components/Apis/Details/Resources/components/OperationsSelector.jsx

```diff
@@ -52,35 +53,75 @@ export default function OperationsSelector(props) {
             <Grid item />
             <Grid item>
                 <Box mr={17.25} display='flex'>
-                    <Tooltip title='Disable security for all'>
-                        <div>
-                            <IconButton
-                                disabled={isRestricted(['apim:api_create'], apiFromContext)}
-                                onClick={disableSecurity}
-                                aria-label='disable security for all'
-                                size='large'
-                            >
-                                <LockOpenIcon />
-                            </IconButton>
-                        </div>
-                    </Tooltip>
-                    <Tooltip title='Enable security for all'>
-                        <div>
-                            <IconButton
-                                disabled={isRestricted(['apim:api_create'], apiFromContext)}
-                                onClick={enableSecurity}
-                                aria-label='enable security for all'
-                                size='large'
-                            >
-                                <LockIcon />
-                            </IconButton>
-                        </div>
-                    </Tooltip>
+                    { operationWithSecurityCount === 0
+                    && (
+                        <Tooltip
+                            title='Enable security for all'
+                        >
+                            <div>
+                                <IconButton
+                                    disabled={isRestricted(['apim:api_create'], apiFromContext)}
+                                    onClick={enableSecurity}
+                                    aria-label='enable security for all'
+                                    size='large'
+                                >
+                                    <LockOpenIcon />
+                                </IconButton>
+                            </div>
+                        </Tooltip>
+                    )}
+                    { operationWithSecurityCount === operationCount
+                    && (
+                        <Tooltip
+                            title='Disable security for all'
+                        >
+                            <div>
+                                <IconButton
+                                    disabled={isRestricted(['apim:api_create'], apiFromContext)}
+                                    onClick={disableSecurity}
+                                    aria-label='disable security for all'
+                                    size='large'
+                                >
+                                    <LockIcon />
+                                </IconButton>
+                            </div>
+                        </Tooltip>
+                    )}
+                    { operationWithSecurityCount !== 0 && operationWithSecurityCount !== operationCount
```

Comment:
Go with
```
(operationWithSecurityCount !== 0 && operationWithSecurityCount !== operationCount)
                    &&
```
Instead of
```
operationWithSecurityCount !== 0 && operationWithSecurityCount !== operationCount
                    &&
```

Reply:
Fixed with 77a1b82